RELEASE/scripts/autoscend/combat/auto_combat_util.ash

```diff
@@ -57,16 +57,32 @@ boolean canUse(skill sk, boolean onlyOnce)
 		return false;
 	}
 
-	if(my_mp() < mp_cost(sk) - combat_mana_cost_modifier() ||
+	if(inCombat)
+	{
+		if(my_mp() < mp_cost(sk) - combat_mana_cost_modifier() ||
 		my_hp() <= hp_cost(sk) ||
 		get_fuel() < fuel_cost(sk) ||
 		my_lightning() < lightning_cost(sk) ||
 		my_thunder() < thunder_cost(sk) ||
 		my_rain() < rain_cost(sk) ||
 		my_soulsauce() < soulsauce_cost(sk) ||
 		my_pp() < plumber_ppCost(sk))
+		{
+			return false;
+		}
+	}
+	else
 	{
-		return false;
+		if(my_maxmp() < mp_cost(sk) - combat_mana_cost_modifier() ||
```

Comment:
same comment as above. Should add, not subtract

Reply:
done